src/lwc/rd2ElevateInformation/rd2ElevateInformation.js

```diff
@@ -29,14 +31,18 @@ import elevateDisabledMessage from '@salesforce/label/c.RD2_ElevateDisabledMessa
 import elevateRecordCreateFailed from '@salesforce/label/c.RD2_ElevateRecordCreateFailed';
 import commonUnknownError from '@salesforce/label/c.commonUnknownError';
 import viewErrorLogLabel from '@salesforce/label/c.commonViewErrorLog';
+import UpdatePaymentInformation from '@salesforce/label/c.UpdatePaymentInformation';
 
 import getData from '@salesforce/apex/RD2_ElevateInformation_CTRL.getData';
 
 const FIELDS = [
     FIELD_NAME,
+    FIELD_PAYMENT_METHOD,
```

Comment:
This may also be a conflict with PR #6242. See the changes I made this f[ile](https://github.com/SalesforceFoundation/NPSP/blob/7f3e4feba07d741a72979e3638507d4d9711e43d/src/lwc/rd2ElevateInformation/rd2ElevateInformation.js#L41).

Reply:
There will probably be a merge conflict after your ticket is merged. However, the change will be minimum since the different just 2 extra fields to query in `FIELDS`